view/rooms/index.pug

```diff
@@ -0,0 +1,35 @@
+//
+  Copyright (c) 2017, salesforce.com, inc.
+  All rights reserved.
+  Licensed under the BSD 3-Clause license.
+  For full license text, see LICENSE.txt file in the repo root or
+  https://opensource.org/licenses/BSD-3-Clause
+
+doctype html
+html
+  head
+    title Refocus Room
+    link(rel='shortcut icon', href='favicon.ico')
+    link(rel='stylesheet', type='text/css', href='../static/css/salesforce-lightning-design-system.min.css')
+    script(src='http://code.jquery.com/jquery-1.11.1.js')
```

Comment:
also - can you remind me why do we need jquery?

Reply:
@kamilsmuga ive updated the resources to use the secure version; the bdk and bots use/may use jquery respectively